resources/js/components/ShowRecords/ShowRecordsComponent.vue

```diff
@@ -0,0 +1,104 @@
+<template>
```

Comment:
New component to show imported records. As a **future** improvementcwe can consider the use of the `vue router`